pkg/apis/v1alpha1/ecsnodeclass_status.go

```diff
@@ -16,6 +16,16 @@ limitations under the License.
 
 package v1alpha1
 
+import (
+	"github.com/awslabs/operatorpkg/status"
+)
+
+const (
+	ConditionTypeVSwitchsReady        = "VSwitchsReady"
+	ConditionTypeSecurityGroupsReady  = "SecurityGroupsReady"
+	ConditionTypeInstanceProfileReady = "InstanceProfileReady"
```

Comment:
I didn't see alibaba cloud use instance profile, they use different things: https://www.alibabacloud.com/help/en/ecs/user-guide/attach-an-instance-ram-role-to-an-ecs-instance

cc @IrisIris to confirm.

Maybe we shoud ignore this becuase we use aksk temporarily.

Reply:
Let's use `InstanceRAMReady`